src/components/Header.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import profilePicture from "../images/profile.jpg"
+import emailIcon from "../images/Mail.png"
+import linkedInIcon from "../images/linkedin.png"
+
+export default function Header() {
+    return (
+        <header>
+        <div className="profile--image">
+            <img src={profilePicture} alt="" />
+        </div>
+        <h1 className="profile--title">Sebastian Wantulok</h1>
```

Comment:
Zwykle -- oznacza klasę modyfikator, u Ciebie jest to tak naprawdę oddzielenie elementu podrzędnego od nadrzędnego - sugeruję użyc podkreślenia albo pojedynczego -